.eslintrc.json

```diff
@@ -0,0 +1,15 @@
+{
+    "env": {
+        "browser": true,
+        "es2021": true
+    },
+    "extends": [
+        "airbnb-base"
+    ],
+    "parserOptions": {
+        "ecmaVersion": 12,
+        "sourceType": "module"
+    },
+    "rules": {
+    }
```

Comment:
- [ ] OPTIONAL - Ensure you use the same eslint rules stated in this [link](https://github.com/microverseinc/linters-config/tree/master/javascript)